src/extensions/score_metamodel/checks/attributes_format.py

```diff
@@ -56,16 +56,23 @@ def check_id_length(app: Sphinx, need: NeedsInfoType, log: CheckLogger):
     While the recommended limit is 30 characters, this check enforces a strict maximum
     of 45 characters.
     If the ID exceeds 45 characters, a warning is logged specifying the actual length.
+    Any examples that are required to have 3 parts (2x'__') have an exception, and get 16 extra characters
+    to compensate for the lenght of `example_feature` that would be replaced by actually feature names.
     ---
     """
-    if len(need["id"]) > 45:
+    max_lenght = 45
+    parts = need["id"].split("__")
+    if parts[1] == "example_feature":
+        max_lenght += 15 # '_example_feature_'
```

Comment:
```suggestion
        max_lenght += 15 # 'example_feature'
```
Forgot to change it here too, I only did it in the desc. Sorry